src/pages/ProductPage.js

```diff
@@ -0,0 +1,80 @@
+import { useState, useEffect } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+import { GetData } from "../API";
```

Comment:
If GetData is not a component, better to name it with first letter in lowercase because its a convention in react to keep the first letter caps for components only

Reply:
didnt knew that thank you